CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 ### Added
 
 - We added a field showing the BibTeX/biblatex source for added and deleted entries in the "External Changes Resolver" dialog. [#9509](https://github.com/JabRef/jabref/issues/9509)
+- We added an option in Groups Preferences to set the default hierarchical context used when creating a new group [#9141](https://github.com/JabRef/jabref/issues/9141)
```

Comment:
Can you please update the changelog entry? And maybe put it under Changed instead of Added.